lib/SILGen/SILGen.h

```diff
@@ -147,6 +147,12 @@ class LLVM_LIBRARY_VISIBILITY SILGenModule : public ASTVisitor<SILGenModule> {
   SILFunction *getDynamicThunk(SILDeclRef constant,
                                CanSILFunctionType constantTy);
 
+  // SWIFT_ENABLE_TENSORFLOW
+  /// Get the autodiff associated function thunk for a JVP/VJP SILDeclRef.
+  /// Currently used only for JVP/VJP vtable entries.
+  SILFunction *getAutoDiffThunk(SILDeclRef constant,
```

Comment:
Consider a more specific name (e.g. `getClassMemberAutoDiffThunk`) since this thunk only has one current use?
Note that protocol witnesses are also thunked but don't require this extra thunking entry point (see [`getWitnessFunctionRef`](https://github.com/apple/swift/blob/c1dfcde1a4ddc3a6aad97571aeb38fab5f9ce37d/lib/SILGen/SILGenPoly.cpp#L4147).